### DISCUSSION TITLE
Push Force

### DISCUSSION THREAD

**Discussion by jorge-valdivia-4** [jesseduffield/lazygit#3527] [Q&A] [answered]
There is some way to execute this command from the app
git push origin <some-branch> -f
I can't find any option on menu to force a push

**Comment by stefanhaller** [comment, marked as answer]
You just push (using `shift-P`). Normally, if force-pushing is necessary, lazygit will ask you for confirmation to force-push. No separate command necessary.

Note however that there is a specific workflow where this doesn't work, see #3437. This is still unresolved, I'd be interested if this affects you too.

**Reply by jorge-valdivia-4** [reply]
Thanks for the confirmation , I have been thinking that should work in that way but I prefer send the question to have a double check.
Regards.

**Comment by pimvanderheijden** [comment]
How to disable force push altogether? I've accidentally hit Enter a couple times already and removed some commits that I needed, was not a great experience.

This is the only thing I could find about it, so hence I post my question here.

**Reply by nielsbom** [reply]
That can be configured, see [docs](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md?plain=1#L484)
